styles/_basecoat-footer.scss

```diff
@@ -13,17 +13,26 @@
 }
 
 .footer-nav {
+  
```

Comment:
I'm totally being a stickler here, but did you want to 🔥 this blank line?

Reply:
Fixed! This made me curious about Sass linting. Have folks on the design team experimented with style linting at all?

Currently seeing over 1000 issues using the vanilla [sass-lint](https://ghub.io/sass-lint) config: https://gist.github.com/zeke/bb7abcdab4a95ac702d4486cb6d65d42

Reply:
> Have folks on the design team experimented with style linting at all?

Yup! They have something for Primer that might be interesting for you: https://github.com/primer/primer/tree/master/tools/stylelint-config-primer

> Currently seeing over 1000 issues using the vanilla sass-lint config: 

I'd expect that to happen when introducing rules after the fact, which can be frustrating at times. Something to consider doing is disabling rules that you would not consider important. Another approach to introducing linting is something that my friend does with [Access Lint](https://www.accesslint.com/) is to report regressions (such as only lint code that has changed and not everything at once.)

Reply:
Nice! I went ahead and set up [AccesLint](https://www.accesslint.com/) for this repo. Let's see how it goes.